test/controllers/content_items_controller_test.rb

```diff
@@ -137,15 +137,13 @@ class ContentItemsControllerTest < ActionController::TestCase
 
     content_store_has_item(content_item['base_path'], content_item)
 
-    with_variant EducationNavigation: "A" do
```

Comment:
No, it's just that I incorrectly added the meta tag and response headers before, where they shouldn't be applied if we've not got content tagged to a taxon.